core/tools/generate_names.cc

```diff
@@ -277,9 +277,8 @@ NameDef names[] = {
     // on a class to lookup an associated symbol with some extra info.
     {"sealedSubclasses", "sealed_subclasses"},
 
-    // This name is used as a key in SymbolInfo::members to store the module
-    // registered via the `mixes_in_class_method` name.
-    {"classMethods", "<class methods>"},
+    // Used to store arguments to a "mixes_in_class_methods()" call
+    {"mixedInClassMethods", "<mixes in class methods>"},
```

Comment:
```suggestion
    {"mixedInClassMethods", "<mixed in class methods>"},
```

As to avoid confusion with `mixesInClassMethods` below and make later debugging easier.